src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -523,17 +534,32 @@ private boolean plantSapling(@NotNull final BlockPos location)
                                     || (!job.tree.isSlimeTree() && !Compatibility.isSlimeDirtOrGrass(dirt))))
         {
             final ItemStack stack = getInventory().getStackInSlot(saplingSlot);
-            final Block block = ((ItemBlock) stack.getItem()).getBlock();
             worker.getCitizenItemHandler().setHeldItem(EnumHand.MAIN_HAND, saplingSlot);
 
-            placeSaplings(saplingSlot, stack, block);
-            final SoundType soundType = block.getSoundType(world.getBlockState(location), world, location, worker);
-            world.playSound(null,
-              this.worker.getPosition(),
-              soundType.getPlaceSound(),
-              SoundCategory.BLOCKS,
-              soundType.getVolume(),
-              soundType.getPitch());
+            if (job.tree.isDynamicTree() && Compatibility.isDynamicTreeSapling(stack))
+            {
+                Compatibility.plantDynamicSapling(world,location,stack);
+                new InvWrapper(getInventory()).extractItem(saplingSlot, 1, false);
+                worker.swingArm(worker.getActiveHand());
+                this.getOwnBuilding().getColony().getStatsManager().incrementStatistic("saplings");
```

Comment:
no need to use statsmanager or achievements, at least I deleted it in upcoming advancements PR